include/RAJA/policy/cuda/policy.hpp

```diff
@@ -20,7 +20,7 @@
 
 #include "RAJA/config.hpp"
 
-#if defined(RAJA_ENABLE_CUDA)
+#if (defined(__NVCC__) || (defined(__clang__) && defined(__CUDA__))) && defined(RAJA_ENABLE_CUDA)
```

Comment:
It might be helpful to `#define` something when `(defined(__NVCC__) || (defined(__clang__) && defined(__CUDA__))) && defined(RAJA_ENABLE_CUDA)` and reuse that to avoid having to reuse this longer line

Reply:
Good point, I was just thinking that.

Reply:
What about using __CUDACC__?

Reply:
Sounds good let me fix that

Reply:
I like this PR, but I might even go one step farther toward what @tomstitt suggested and have say `RAJA_CUDA_ACTIVE` or something as an alias for `defined(__CUDACC__) && RAJA_ENABLE_CUDA` so we don't have to remember to write the and op wherever it comes up.

Reply:
That sounds good, I'll add that under the config.hpp.in file and propagate it accordingly.